### ISSUE TITLE
[WASM + iOS] Regression: UNO 5.4 broke the .Tapped event

### ISSUE THREAD

**Issue by DierkDroth** [unoplatform/uno#18599]
### Current behavior

Please see repo below. Here is the scenario:
- on iOS (iPhone or iPad, browser does not matter since it's all WebKit anyway)
- tap any of the rows -> a new UI opens
- close that UI by pressing the button -> brings you back to the original UI
- you **now have to click twice** on a row to again bring up the 2nd UI -> **wrong**, a single click should do

This worked no problem with a single click on UNO WASM 5.3, but no longer works on UNO WASM 5.4 -> you have to click twice.
Note: 
- this is a problem with the touch events. Mouse events work no problem
- there is no problem with Android touch devices (Chrome)

[Minimal.zip](https://github.com/user-attachments/files/17531631/Minimal.zip)

Here are the links to test apps - so you don't have to build and upload:
- [this is the UNO WASM 5.3 version](https://resources.unusual-suite.com/app/test/tapped-working/wwwroot/index.html) -> works
- [this is the UNO WASM 5.4 version](https://resources.unusual-suite.com/app/test/tapped-broken/wwwroot/index.html) -> **broken**

@jeromelaban I was happy as @dr1rrb [fixed this UNO 5.2 regression (with UNO  5.4)](https://github.com/unoplatform/uno/issues/16928) which prevented me from upgrading. Now, I'm hitting the wall again and still can't upgrade my UNO 5.1 solution.

Is there any chance this regression could be addressed short term? If not, is there any work around which I could apply? 
Thanks in advance for considering that regression.

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

**Comment by jeromelaban** [comment]
@DierkDroth we do not have the bandwidth to address this issue at this time, though we may be able to take a look in the coming weeks.

**Comment by DierkDroth** [comment]
Thanks for taking care @jeromelaban

**Comment by DierkDroth** [comment]
@jeromelaban @dr1rrb I still am stuck with my production version on UNO 5.1 - which now is almost 1 year old - due to the cascade of regressions (see above). The last remaining one (I'm aware of) is the issue as per above.

Is there anything which prevents https://github.com/unoplatform/uno/pull/18933 from getting merged?
Is there any work around/configuration change/... I could apply to mitigate the regression?
Is there anything I could help with to get this regression ironed out?

**Comment by francoistanguay** [comment]
We should be able to look at it in upcoming days/weeks.

**Comment by DierkDroth** [comment]
@dr1rrb I just tested the issue in my app using latest UNO 5.6.70. Unfortunately the problem persists.

Is that expected?

**Comment by jeromelaban** [comment]
@DierkDroth yes it's expected, and it is already available in 5.7-dev builds.

**Comment by DierkDroth** [comment]
Thanks for clarification @jeromelaban

**Comment by DierkDroth** [comment]
@jeromelaban this still doesn't work (reliably) with latest UNO 5.7.0-dev.660. Is that expected?

**Comment by jeromelaban** [comment]
It depends on what you mean by reliably does not work. At this time, this is the only fix we have. If you can provide additional details into what you're seeing, we may be able to look at it again.

**Comment by dr1rrb** [comment]
Hey @DierkDroth are you able to give more details about your issue? When testing different scenarios with https://aka.platform.uno/wasm-samples-app on my phone, I'm unable to repro any issue.

**Comment by DierkDroth** [comment]
@dr1rrb sure, I wanted to follow up anyway. However, I'm busy with other stuff right now. Nevertheless, I will get back to you guys as soon as I can...

**Comment by dr1rrb** [comment]
Ok sure no stress, thanks for your help @DierkDroth

**Comment by DierkDroth** [comment]
@dr1rrb I updated my repo above to latest UNO and no longer can reproduce the issue.

However, the issue still is there in my actual app. May be to a lesser degree - I'm not sure. It might help if I granted you access to the DEV version of my app so we could try to isolate the remaining source of trouble. Could you please PM me at dierk DOT droth AT unusual-suite DOT com so I could grant you access to a test account?

**Comment by francoistanguay** [comment]
Hi Dierk,

Unfortunately, we don't fix issues against people's personal repos. A new repro will need to be found so that we could eventually look into it again. 

Thank you
